config/webpack/dev.js

```diff
@@ -1,20 +1,22 @@
+var fs = require('fs');
 var path = require('path');
 var webpack = require('webpack');
 var postcssAssets = require('postcss-assets');
 var postcssNext = require('postcss-cssnext');
 var stylelint = require('stylelint');
 var ManifestPlugin = require('webpack-manifest-plugin');
 
+const {
```

Comment:
No worries. I should've ran the tests before.

I really dislike the need to manage multiple copies/versions of the webpack configs, 
it's cumbersome and error prone. These guys did quite an amazing job, it's used in Aurelia: https://github.com/easy-webpack
Perhaps it's interesting to look into.